Fix configuring only stateboard instance (#269)
tarantool/ansible-cartridge: doc/scenario.md

  - `instance_dist_dir` - path to instance link to distributed package;
- `single_instances_for_each_machine` - list of instances (Ansible hosts), one for each physical machine. Can be used,
  for example, in `delegate_to`;
- `scenario_steps` - description of scenario steps. Each step is a dictionary with fields:
  - `name` - name of step;
  - `path` - path to YAML file of step;

> There are other facts that are set during role execution or intialized with values

Reviewer: What about `package_info` e.t.c. These are facts that are also established in the process of performing the role.

set by user:

- `control_instance` - information about control instance, that should be used for
  managing topology and configuring cluster. It's a dictionary with fields:
  - `name` - instance name (Ansible host);
  - `console_sock` - path to control socket of instance.

  This fact is set on role preparation if `cartridge_control_instance` is specified.
  This fact can be set explicitly using [`set_control_instance` step](#set_control_instance).
  Steps that require control instance (such as [`edit_topology`](#edit_topology))
  call `set_control_instance` implicitly if `control_instance` fact isn't set.

- `delivered_package_path` - remote path to file of delivered package.
  Is set on role preparation if `cartridge_delivered_package_path` is specified.

>   This fact is set up on [`deliver_package` step](#deliver_package).

Reviewer: ```suggestion
  Also, this fact will be set up in [`deliver_package` step](#deliver_package).
```

  - `instance_dist_dir` - path to instance link to distributed package;
- `single_instances_for_each_machine` - list of instances (Ansible hosts), one for each physical machine. Can be used,
  for example, in `delegate_to`;

> - `not_expelled_instance` - information about one not expelled instance. It's a dictionary with fields:

Reviewer: You should add information about this variable in connect_to_membership and set_control_instance steps
